docs/tutorials/json_schema.md

```diff
@@ -0,0 +1,163 @@
+In this tutorial, you will learn how to bind a JSON Schema to a folder. You can also bind a JSON Schema to a project, a file, a table, or a view. This tutorial will walk you through each step, from schema creation to validation.
```

Comment:
There should be a short explanation on what JSON Schema is

Reply:
Thanks for the suggestion, Bryan! I borrowed the explanation from the synapse docs.